flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/utils/TestTableSourceFactoryBase.java

```diff
@@ -100,7 +100,7 @@ public TestTableSourceFactoryBase(String type, String testProperty) {
 	public StreamTableSource<Row> createTableSource(TableSourceFactory.Context context) {
 		TableSchema schema = context.getTable().getSchema();
 		final DescriptorProperties params = new DescriptorProperties(true);
-		params.putProperties(context.getTable().getProperties());
+		params.putProperties(context.getTable().toProperties());
```

Comment:
Why do you change this? Is it a mistake?

Reply:
`SchemaValidator.deriveRowtimeAttributes` need properties contains `TableSchema`.

Reply:
The class was `createTableSource(properties)`, we should revert to that properties.

Reply:
Thank you. I see.